database/migrations/2022_06_06_220223_add_new_tags_v2.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddNewTagsV2 extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('alcohol', function (Blueprint $table) {
+            $table->integer('cup')->unsigned()->nullable();
+            $table->integer('packaging')->unsigned()->nullable();
+        });
+        Schema::table('coastal', function (Blueprint $table) {
+            $table->integer('degraded_bottle')->unsigned()->nullable();
+            $table->integer('degraded_bag')->unsigned()->nullable();
+        });
+        Schema::table('food', function (Blueprint $table) {
+            $table->integer('packaging')->unsigned()->nullable();
+            $table->integer('cutlery')->unsigned()->nullable();
```

Comment:
These new additions are welcome, but I think we should keep the original ones too, as it enables the user to select the material + item at once. Will see what the group thinks